### ISSUE TITLE
My VKTRS flow errors out at the Initial Audio Processing stage 

### ISSUE THREAD

**Issue by ghost** [dmarx/video-killed-the-radio-star#158]
First off, I am actually NOT a developer.  I am NOT technical.  That said, I really want to create some AI videos to accompany some of my better songs (I've been writing and recording for over 20 years). 

I am working my way through the VKTRS workbook and everything has been smooth until I received an unexpected error during Initial Audio Processing.  Any idea what is going on here?  

**FileNotFoundError: [Errno 2] No such file or directory: '/content/drive/MyDrive/AI/VideoKilledTheRadioStar/shared_assets/DOWNLOADED__Active Joy.json'**

I double checked, and sure enough this .JSON file isn't present in the expected Gdrive directory location. 

Any way you can help me proceed to the next level of VKTRS awesomeness?

Regards,
Jason

**Comment by npgilbertson** [comment]
I got the same error. I ended up clicking on the left side folder icon i deleted the "models" and "Video Killed The Radio Star" folders.
Looks like it has something to do in the code with it seeing the audio file in the folder (like if you manually put one in there) and skipping making the .json file entirely. I dont know enough to fix it fully though sadly, so it seems restarting like that works.

TLDR:
Delete your "models" and "Video Killed The Radio Star" Folders

**Comment by npgilbertson** [comment]
I havent seen if the rest works but so far 
_Replace the code in "Install dependencies with:_
```
 #%%capture

# @title # 🛠️ Setup: Install Dependencies

# Install dependencies

try:
    import google.colab
    local=False
except:
    local=True

# TODO: pin versions

# local only additional dependencies
if local:
    %pip install pandas torch pillow beautifulsoup4 scipy toolz numpy lxml librosa scikit-learn rich

# dependencies for both colab and local
%pip install torch==2.0.1 triton==2.0.0 yt-dlp python-tsp stability-sdk[anim_ui] diffusers transformers ftfy accelerate omegaconf
%pip install openai-whisper panel huggingface_hub ipywidgets safetensors keyframed demucs parse""
```

Is at least getting me past that first stage. Basically the issue isn't the json file at all. It's before that, in that triton V2.1 doesn't work with another guy in there. Causing cuda.so to not be found. Code tries to continue (should have an error system for anything file related imo to break the code) and tries to find it's not yet made json because it failed earlier. I also made an empty .json file with just 
[
]
in the file, but thats not needed with the above code!

**Comment by PandaShape** [comment]
I have the same issue, even with the replacement of install dependencies
